### PR TITLE
start of pxe role, sown/tasks#62

### PR DESCRIPTION
**Pull request by TimStallard**
Only half tested on GW2 for now, need to split DHCP range and try on GW, will probably have a go tomorrow.

### CODE REVIEW

File: roles/pxe/tasks/main.yml [hunk 1]
```diff
@@ -0,0 +1,2 @@
+- import_tasks: dnsmasq.yml
```
Comment (trickeydan) [inline roles/pxe/tasks/main.yml]: tags pls

File: roles/pxe/tasks/main.yml [hunk 2]
```diff
@@ -0,0 +1,2 @@
...
+- import_tasks: ipxe.yml
```
Comment (trickeydan) [inline roles/pxe/tasks/main.yml]: tags pls

File: roles/pxe/templates/dnsmasq.conf.j2 [hunk 3]
```diff
@@ -0,0 +1,18 @@
+#disable dns server
```
Comment (trickeydan) [inline roles/pxe/templates/dnsmasq.conf.j2]: ??

File: roles/pxe/handlers/main.yml [hunk 4]
```diff
@@ -0,0 +1,4 @@
+- name: Reload dnsmasq
```
Comment (trickeydan) [inline roles/pxe/handlers/main.yml]: trailing whitespace

File: host_vars/GW.yml [hunk 5]
```diff
@@ -0,0 +1,2 @@
+dhcp_range_start: "10.5.0.128"
```
Comment (trickeydan) [inline host_vars/GW.yml]: We might have a DHCP role later, would recommend `pxe_dhcp_....`

File: roles/pxe/handlers/main.yml (hunk 4 above)
Comment (TimStallard) [reply roles/pxe/handlers/main.yml]: argh, I fixed that last night and git diff wasn't showing it so I rolled back :/

File: roles/pxe/templates/dnsmasq.conf.j2 (hunk 3 above)
Comment (TimStallard) [reply roles/pxe/templates/dnsmasq.conf.j2]: we use BIND as a slave on the routers, so don't use dnsmasq's dns server

Comment (TimStallard) [reply roles/pxe/templates/dnsmasq.conf.j2]: (will have a role for the bind slave bit shortly :)

**Pull request merged**